### ISSUE TITLE
You are wrongly redirected to the "Select an email template" page even if you are not logged in to "Gmail"

### ISSUE THREAD

**Issue by Softvision-MariusComan** [mozilla/email-tabs#162]
**[Affected versions]:**
- Firefox 62.0.2 and above

**[Affected Platforms]:**
- All Windows
- All Mac
- All Linux

**[Prerequisites]:**
- Have a Firefox profile with the latest version of the "Email Tabs" add-on (0.1.6560) custom built on 10/2/2018 installed.
- You are not logged in to "Gmail".

**[Steps to reproduce]:**
1. Open the browser with the profile from prerequisites and navigate to "youtube.com".
2. Click the "Email Tabs" toolbar button.
3. Click the "Email Tabs" button and observe the behavior.

**[Expected result]:**
- The "Gmail" login page is opened.

**[Actual result]:**
- The "Gmail" login page is opened for a short period of time and you are automatically redirected to the "Select an email template.

**[Notes]:**
- This issue is reproducible with the following websites:
	- https://www.youtube.com/
	- https://www.reddit.com/
	- https://www.twitch.tv/
- This issue is not reproducible with the following websites:
	- https://www.facebook.com/
	- https://www.wikipedia.org/
- Also if you will select an email template the "Preparing your email..." page will be displayed indefinitely.
- Attached a screen recording of the issue:
![rec of the issue](https://user-images.githubusercontent.com/31243259/46421563-0c68e900-c73b-11e8-88f6-1260ccd05470.gif)

**Comment by johngruen** [comment]
We should fix this and show the warning notification when we bail out.

**Comment by Softvision-PatriciuPop** [comment]
I have retested this issue and it is still reproducible using email_tabs-0.1.7232 on Windows 10 x64, Mac 10.13 and Arch Linux using Firefox Nightly Version 65.0a1, Build ID 20181028102553. 
It looks like the actual result has changed. Now, the "Gmail" login page is no longer opened for a short period of time and you are automatically redirected to the "Select an email template". 

I have attached a screen recording and considering this I will reopen the issue.
![162](https://user-images.githubusercontent.com/40393891/47653142-7d5fbd00-db90-11e8-85aa-89a638a8372f.gif)
